API.md

```diff
@@ -49,6 +49,7 @@ Additional configuration keys that can be passed to `auth()` on initialization:
 - **`redirectUriPath`** - Relative path to the application callback to process the response from the authorization server. This value is combined with the `baseUrl` and sent to the authorize endpoint as the `redirectUri` parameter. Default is `/callback`.
 - **`required`** - Use a boolean value to require authentication for all routes. Pass a function instead to base this value on the request. Default is `true`.
 - **`routes`** - Boolean value to automatically install the login and logout routes. See [the examples](EXAMPLES.md) for more information on how this key is used. Default is `true`.
+- **`tokenAuthMethod`** - String value for the expected token auth method. Default is taken from Issuer supported methods.
```

Comment:
```suggestion
- **`tokenAuthMethod`** - String value for the token endpoint authentication method. Default is `client_secret_basic`.
```

Reply:
The point here is that auth method should not be automatically selected based on an optional issuer property. There is a default used that can be overriden, that's fine, but there's no way to "select" the one that should be used from an unprioritized, unordered list.